### ISSUE TITLE
NTR: Ghrelin high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8425]
New term request Ghrelin high in blood (elthat_190101175840):   UMLS: 4748438

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000436
- label: Elevated circulating ghrelin concentration
- parents: [Abnormal circulating hormone concentration (HP:0003117)](https://hpo.jax.org/app/browse/term/HP:0003117)
- pmid: PMID:29632576
- def: The concentration of ghrelin in the blood circulation is above the upper limit of normal. Ghrelin is a 28-amino-acid peptide predominantly secreted in the stomach and stimulates appetite and growth hormone (GH) release.
- comment: Ghrelin exercises a wide range of functions including, regulation of food intake and energy metabolism, stimulation of gastric acid secretion, motility and pancreatic protein output, modulation of cardiovascular function, stimulation of osteoblast proliferation and bone formation, stimulation of neurogenesis and myogenesis, learning and memory, thymopoiesis, sleep/wake rhythm, ageing and a neuroprotective role in neurodegenerative diseases (e.g., Parkinson's disease).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8425
- synonyms:
